### ISSUE TITLE
Composer should run npm install after running

### ISSUE THREAD

**Issue by AaronHolbrook** [eugene-manuilov/wp-codeception#6]
Need to have the node_modules available.

That, or we should include them in the repo

**Comment by AaronHolbrook** [comment]
It looks like composer won't run scripts that are not in the main package, which means I think our only real option is to simply include them in the repo.

I'm definitely open to other ideas should anyone have them.

**Comment by AaronHolbrook** [comment]
Hmm - the other thing the `npm install` command does is link phantomjs. Need to find a way to do that from within our wp-cli command if possible.

**Comment by fuhton** [comment]
@AaronHolbrook setting this up and running into the same issues if PhantomJS isn't globally available. 

Edit: as a followup, it looks like the most recent version of phantomJS (1.9.18) was causing some issues. Switching to 1.9.16 resolved any issues I was having.

**Comment by jonathanbardo** [comment]
What about using a `post-install-cmd`?

```
"scripts": {
    "post-install-cmd": [
      "bin/install-wp-codeception.sh"
    ]
  }
```

And in install-wp-codeception.sh: 

``` bash
#!/usr/bin/env bash
sudo apt-get install openjdk-7-jre-headless --yes
cd vendor/wp-codeception
npm install
```

**Comment by eugene-manuilov** [comment]
Good idea, @jonathanbardo! Please, create a pull request and I'll add it.

**Comment by jonathanbardo** [comment]
@eugene-manuilov Actually I think this needs to go in the package that requires wp-condeception like I did here https://github.com/jonathanbardo/WP-Contact-Widgets/blob/develop/composer.json
